.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@MerrybyPractice Can you pul .vscode out and maybe add it to .gitignore? It has a setting that is specific to your machine.

Reply:
@marvincolgin, so sorry about that! Updated. Let me know if that did not resolve the issue.